skimage/measure/_regionprops.py

```diff
@@ -160,12 +160,12 @@ def __init__(self, slice, label, label_image, intensity_image,
         self._cache = {}
         self._ndim = label_image.ndim
 
-    @property
+    @property  # type: ignore  # type: ignore
```

Comment:
typo?

Also, same as above, why do `@property` decorators need to be ignored? It seems like a pretty major issue that MyPy would have had to have addressed by now, surely?

Reply:
See https://github.com/python/mypy/issues/1362. People are still linking to this issue as recently as Dec 2019 so I'm not sure it's fixed. Let me take a closer look.

Reply:
and sure this was a search and replace typo